dj_local_conf_template.json

```diff
@@ -11,6 +11,6 @@
     "display.width": 14,
     "display.show_tuple_count": true,
     "history": [],
-    "do_unittest": true,
+    "do_unittest": false,
     "custom": {}
-}
```

Comment:
This `dj_local_conf_template.json` can and should be removed. Meant to serve as a template for new users, but seems not very useful now

Reply:
Agreed - let's get rid of this.